github-rate-limits.py

```diff
@@ -18,31 +18,18 @@
     print("Max calls: ", limit)
     reset_time = datetime.fromtimestamp(gh.rate_limiting_resettime)
     print("Reset time (GMT): ", reset_time)
-
-    JENKINS_PREFIX = "jenkins"
-    try:
-        JENKINS_PREFIX = os.environ["JENKINS_URL"].strip("/").split("/")[-1]
-    except:
-        JENKINS_PREFIX = "jenkins"
+    gh_user = gh.get_user("cms-sw")
```

Comment:
this is one api call, is there any other way to get user with making an API call?

Reply:
Mmm... I don't see any property for getting the user. There is only one more apart from the `rate_limiting` and `rate_limiting_resettime`, but it doesn't return any user info ([oauth_scopes](https://pygithub.readthedocs.io/en/latest/github.html#github.MainClass.Github.oauth_scopes)):
```
['admin:org', 'gist', 'repo', 'user', 'workflow']
```

Reply:
Using the local linux user name instead :)